### ISSUE TITLE
HPC scheduler is LSF :(

### ISSUE THREAD

**Issue by benyoung93** [jonahcullen/wags#33]
Good morning :). 

First of all this tool looks extremely impressive and will allow streamlining of our pipelines that currently have way to main different jobs. 

Unfortunately, the current HPC I have access to uses LSF for scheduling (sighhhhh). As I am a wee bit new to snakemake, how hard would it be to generate a --profile for LSF? IS this something you would have the time to do ?

If it is simple and there is a way for me to do it please let me know, any linking of articles/protocols for it I can give it a go. 

Ben

**Comment by jonahcullen** [comment]
Hey Ben, I'm happy to try and generate a LSF profile for you (and whoever else lands here!). The only reason I have not yet as I don't have a quick way of checking it's completeness. So if you wanted to be the proverbial _guinea pig_ I can let you know when I have a draft profile up and ready?

**Comment by benyoung93** [comment]
Yes that sounds wonderful !!! thank you so much for the quick response. I am pretty free with work schedule so whenever you get this up I can test pretty quickly :). 

Ben

**Comment by jonahcullen** [comment]
Can you run `grep '^LSF_UNIT_FOR_LIMITS' ${LSF_ENVDIR}/lsf.conf` from your HPC?

**Comment by benyoung93** [comment]
Hi hi 

Yes of course, output is as follows 

`LSF_UNIT_FOR_LIMITS=MB`

I did have a quick half an hour attempt at this https://github.com/Snakemake-Profiles/lsf which also asked for the command you pasted above. I successfully created a profile but could not get it to work with wags :(

**Comment by jonahcullen** [comment]
Hey @benyoung93 I did not forget about this! I'll get back to you yet this week for you to test it at your institution.

**Comment by jonahcullen** [comment]
Hey again @benyoung93 I unfortunately did _forget_ as I was finishing my thesis and preparing for the final defense. This is back on my radar now. No worries if you moved on, I'll still put it together if you want to test it out later!
